apps/challenges/utils.py

```diff
@@ -189,3 +195,25 @@ def create_federated_user(name, repository, aws_keys):
         DurationSeconds=43200,
     )
     return response
+
+
+@if_non_prod_aws_mocker(mock_ecr)
+@if_non_prod_aws_mocker(mock_sts)
+def get_aws_credentials_for_submission(challenge, participant_team):
```

Comment:
Add some documentation to the function to give a bit more context. Also in the documentation, you can specify mock_sts is for the mocking the creation of the federated user. This will be useful for someone who is working on this later.